Wurst Client for MC 1.9.X/src/tk/wurst_client/mods/AutoMineMod.java

```diff
@@ -33,9 +33,11 @@ public void onUpdate()
 			|| mc.objectMouseOver.getBlockPos() == null)
 			return;
 		if(Block.getIdFromBlock(mc.theWorld.getBlockState(
-			mc.objectMouseOver.getBlockPos()).getBlock()) != 0)
-			mc.gameSettings.keyBindAttack.pressed = true;
-		else
+			mc.objectMouseOver.getBlockPos()).getBlock()) != 0){
+			if(wurst.mods.autoToolMod.isActive())
+				AutoToolMod.setSlot(mc.objectMouseOver.getBlockPos());
+				mc.gameSettings.keyBindAttack.pressed = true;
```

Comment:
Are you sure?

> if(wurst.mods.autoToolMod.isActive())
 +				AutoToolMod.setSlot(mc.objectMouseOver.getBlockPos());

http://stackoverflow.com/questions/15786949/is-there-a-difference-in-removing-the-curly-braces-from-if-statements-in-java/15786982#15786982